encode_test.go

```diff
@@ -9,6 +9,7 @@ import (
 	"time"
 
 	"github.com/lib/pq/oid"
+	"reflect"
```

Comment:
Build is failing on formatting rules. Move this to the group of imports around L6.